package.json

```diff
@@ -8,18 +8,18 @@
   },
   "main": "lib/index.js",
   "scripts": {
-    "test": "npm run lint && npm run test-functional",
-    "lint": "jscs `find ./lib -name '*.js'` -v && jscs `find ./test -name '*.js'` -v",
-    "test-functional": "tap `find ./test/functional -name '*.test.js'`",
-    "test-system": "tap `find ./test/system -name '*.test.js'`",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "lint": "npm run eslint",
+    "eslint": "./node_modules/.bin/eslint -c .eslintrc lib test",
```

Comment:
doesnt it work just with `eslint`, without the relative path? as done with `tap` below

Reply:
yeah totally, will change